src/components/va-featured-content/va-featured-content.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Component,
+  Host,
+  Prop,
+  h,
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-featured-content',
+  styleUrl: 'va-featured-content.css',
+  shadow: true,
+})
+export class VaFeaturedContent {
+  /**
+   * Sets the level for the HTML section heading elements.
+   * Valid values: 3, 4
+   */
+  @Prop() level: number;
+
+  /**
+   * Sets the text for the section heading.
+   */
+  @Prop() header: string;
```

Comment:
Instead of having to use two props, we can use a named slot like this:

https://github.com/department-of-veterans-affairs/component-library/blob/ccd5f49b327edfbfb814153be5a75b4e45257161/src/components/va-alert/va-alert.tsx#L144

This would allow the component to be used like this:

```html
<va-featured-content>
   <h3 slot="headline">Feature headline</h3>
   <div>Body text</div>
</va-featured-content>
```

Reply:
@bkjohnson I read somewhere, that the Featured Content headline can only be `h3` or `h4`. Should we trust that only `h3` or `h4` headlines will be passed to that slot or should I add some sort of error handling like I did for the `level` prop?

See `src/components/va-featured-content/va-featured-content.tsx`:
```
    if (this.level !== 3 && this.level !== 4) {
      throw new Error('Featured Content header level must be 3 or 4');
    }
```

Thanks!

Reply:
I removed the props and updated the tests.

Reply:
I'm not too worried about that being a problem right now. If it _is_ something we want to enforce, we can solve for that later.